general-standards/README.md

```diff
@@ -1,24 +1,76 @@
+- [ ] Format and lint code
+- [ ] Run tests locally
+- [ ] Ensure CI tests pass
+- [ ] Ensure commit messages are succinct
+- [ ] Check and resolve merge conflicts
+- [ ] Review PR in GitHub UI, this makes it much easier to check for miscellaneous print statements, debugging code, and any other silly mistakes
+- [ ] Remove any dead or unused code
+- [ ] Make sure you update `.env.example`, `requirements.txt`, etc. when the environment changes
+- [ ] Check all spelling and grammar within code
+- [ ] Ensure the variables have appropriate names, following convention
+- [ ] Does the PR reference the issue being resolved (e.g. `-#XYZ`)?
+- [ ] Flag any issues in the code that are intentional (e.g. “wontfix for this pr”) or that you are aware of
+
+### UI
+
+- [ ] Read through initial issue, ensure everything is implemented
+- [ ] Have you gone through each desired feature and ensured it functions properly?
+- [ ] Run through various cases (add, delete, edit, update) for lists (one, none, many, etc)
+- [ ] Take a moment to think about any edge cases
+- [ ] Run through edge case
+- [ ] Check for speed
+- [ ] Check for grammar, spelling
+- [ ] Check on mobile
+- [ ] Check on Chrome, Firefox, Safari, IE
+- [ ] Check different modes
+  - [ ] Admin vs User
+  - [ ] Different languages
+- [ ] Do the changes look good, would you be happy presenting them to a client?
+- [ ] Do you fully sign off on these changes?
+- [ ] Does the design meet the standards of the design guide?
+
 ## "AlliedCrowds"
 
 AlliedCrowds should always be styled "AlliedCrowds"
 
 ## Content
 
-* Capitalize only the first word in buttons, texts, descriptions, placeholders, table headers, etc.
-* If the word/phrase is a title, capitalize all words.
-* Acronyms such as URL should be all capitalized, no periods (e.g. UNDP not U.N.D.P. or undp).
-* The case of database records  should be preserved, respect the entity's capitalization standards, e.g., "GlobalGiving"
-* References to methods, functions, database tables and any code should be written exactly as specified in the code and should be referenced using backticks ``.
+- Capitalize only the first word in buttons, texts, descriptions, placeholders, table headers, etc.
+- If the word/phrase is a title, capitalize all words.
+- Acronyms such as URL should be all capitalized, no periods (e.g. UNDP not U.N.D.P. or undp).
+- The case of database records should be preserved, respect the entity's capitalization standards, e.g., "GlobalGiving"
+- References to methods, functions, database tables and any code should be written exactly as specified in the code and should be referenced using backticks ``.
 
 ## Tests
 
-* All new features, releases, hotfixes, and bug fixes must have associated tests or they will not be merged in.
+- All new features, releases, hotfixes, and bug fixes must have associated tests or they will not be merged in.
```

Comment:
What are the caveats for this? Quite often i'm not writing tests for PR but this makes it sounds like I should be doing it all the time? Do we assume that if it's something like a `capital finder` 
 or a `allied labs` implementation then it's not necessary or do you want me to start writing tests for them? I feel definitely anything on the backend should have tests written.